addons/pager/jquery.tablesorter.pager.js

```diff
@@ -328,7 +328,11 @@
 				if (p.fixedHeight && !p.isDisabled) {
 					h = $.data(table, 'pagerSavedHeight');
 					if (h) {
-						d = h - $b.height();
+						var bs = 0;
+						if($(table).css('border-spacing').split(" ").length>0){
+							bs = $(table).css('border-spacing').split(" ")[0].replace(/[^-\d\.]/g, '');
```

Comment:
The second `border-spacing` value should be used here because it contains the horizontal size ([ref](https://developer.mozilla.org/en-US/docs/Web/CSS/border-spacing))

```css
/* border-spacing: horizontal vertical */
border-spacing: 1cm 2em;
```